src/Controller/CourseController.php

```diff
@@ -2,61 +2,89 @@
 
 namespace App\Controller;
 
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
+/**
+ * @Route("/api/courses", name="api_courses_")
+ */
 class CourseController extends AbstractController
 {
-    var $courses = array();
+    /**
+    * @Route("", name="list", methods={"GET"})
```

Comment:
Propozcja, "/" jako route path mógłby być bardziej intuicyjny niż empty string

Reply:
Może w sumie zostawię pełne routy w annotacji zamiast prefiksować cały kontroler dla większej czytelności? Będzie się wam to wygodniej czytać? @MichalDaniluk  @kromacie ?

Reply:
Please write all comments in English